packages/block-library/src/archives/edit.js

```diff
@@ -14,13 +14,11 @@ import { __ } from '@wordpress/i18n';
  */
 import {
 	InspectorControls,
-	BlockAlignmentToolbar,
-	BlockControls,
 	ServerSideRender,
 } from '@wordpress/editor';
 
 export default function ArchivesEdit( { attributes, setAttributes } ) {
-	const { align, showPostCounts, displayAsDropdown } = attributes;
```

Comment:
Is this something we could extract separately? Can we not implement supports.align for server-registered blocks currently?